diff --git a/linux.cf b/linux.cf
index f1ef89a..ffa1e55 100644
--- a/linux.cf
+++ b/linux.cf
@@ -588,11 +588,7 @@ InstallNamedTargetNoClobber(install,file.ad,$(INSTAPPFLAGS),$(XAPPLOADDIR),class
 #define MkdirHierCmd		mkdir -p

 #ifndef HaveLib64
-# if defined (AMD64Architecture) || defined (s390xArchitecture) || defined (Ppc64Architecture) || defined (AArch64Architecture) || defined (RV64Architecture)
-#  define HaveLib64	YES
-# else
 #  define HaveLib64	NO
-# endif
 #endif

 #if UseElfFormat
@@ -1118,7 +1114,7 @@ InstallNamedTargetNoClobber(install,file.ad,$(INSTAPPFLAGS),$(XAPPLOADDIR),class

 #if HaveLib64
 # ifndef LibDirName
-#  define LibDirName		lib64
+#  define LibDirName		lib
 # endif
 # ifndef SystemUsrLibDir
 #  define SystemUsrLibDir	/usr/lib64
